src/nurse/tests.py

```diff
@@ -264,8 +273,7 @@ def test_create_prescription(self):
         response = self.client.post(self.url, self.data, format="json")
         assert response.status_code == status.HTTP_201_CREATED
         assert Prescription.objects.count() == 1
-        prescription = Prescription.objects.get(carte_vitale="12345678910")
-        assert prescription.caisse_rattachement == "12345678910"
+        prescription = Prescription.objects.order_by("-id").first()
```

Comment:
Since we only have one anyway, I think you could simplify with:
```python
prescription = Prescription.objects.get()
```

Reply:
oki I'll update